### PR TITLE
Ana/replace connection module for extension

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #ISSUE

**Description:**

The goal of this PR is to fix the browser extension. I haven't tested it yet.

Will fix the test after the meeting :wink:

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by Bitcoinera** [comment]
Ok, the linter messed up with my props :laughing:

File: src/components/network/NetworkItem.vue [hunk 1]
```diff
@@ -50,7 +47,8 @@ export default {
     }
   },
   computed: {
-    ...mapState([`connection`])
+    ...mapGetters([`connected`]),
```
Comment (faboweb) [inline src/components/network/NetworkItem.vue]:
```suggestion
    ...mapGetters([`connected`, `network`]),
```

Comment (Bitcoinera) [reply src/components/network/NetworkItem.vue]: xD the rush to get it done

**Comment by Bitcoinera** [comment]
I just checked and the extension is working with this PR!! :tada: 

Only one thing missing. I can fix it also here... There is a spinning circle in the network selected (since it is not really connected)

**Comment by faboweb** [comment]
@Bitcoinera just set the getter `connected` in the extension to true

**Comment by Bitcoinera** [comment]
> @Bitcoinera just set the getter `connected` in the extension to true

Mmm, the bad thing about this solution is that then we'd need to have a separate commit for the extension... and I prefer the holistic approach of all code being able to work the same for mobile, extension and webapp.

**Comment by faboweb** [comment]
the getter in the store not in the component

**Pull request merged**